src/types/extensions.ts

```diff
@@ -0,0 +1,16 @@
+interface VotingReputationParams {
+  requiredStake: string;
+  minimumStake: string;
+}
+
+export interface CreateExtensionInput extends Record<string, any> {
+  colonyId: string;
+  hash: string;
+  version: number;
+  installedBy: string;
+  installedAt: number;
+  isDeprecated: boolean;
+  isDeleted: boolean;
+  isInitialized: boolean;
+  extensionConfig: VotingReputationParams | null;
+}
```

Comment:
This looks like a potential maintenance hell. I'd rather keep it as `any` until we get codegen in place.
If we don't update those when the fields change, the mutation will fail regardless, and I feel like all we get from it is a sense of false security